test/Cover/DefaultCoverGeneratorTest.php

```diff
@@ -102,7 +102,7 @@ public function testGetTemplateName()
         $doc->addCollection($subcollection);
         $doc->store();
 
-        $generator = CoverGeneratorFactory::create();
+        $generator = CoverGeneratorFactory::getInstance()->create();
```

Comment:
Das hier ist ja schon eine Art Integrationstest für die `create()` Funktion. Der Test oben ist hier also nicht wirklich notwendig bzw. hier in dieser Testklasse könnte in einer neuen Funktion geprüft werden, ob `DefaultCoverGenerator` zurückgegeben wird. Siehe auch Kommentar zu `CoverGeneratorFactoryTest`.